internal/network/machine_network_cidr.go

```diff
@@ -417,6 +450,57 @@ func GetInventoryNetworksByFamily(hosts []*models.Host, log logrus.FieldLogger)
 	return ret, nil
 }
 
+type CidrMetric struct {
+	cidr   string
+	metric int32
+}
+
+func GetDefaultRouteNetworkByFamily(hosts []*models.Host, log logrus.FieldLogger) map[AddressFamily]string {
+	ret := make(map[AddressFamily]string)
+	cidrmap := make(map[AddressFamily][]CidrMetric)
+	var err error
+
+	for _, h := range hosts {
+		if h.Inventory != "" {
+			var inventory models.Inventory
+			err = json.Unmarshal([]byte(h.Inventory), &inventory)
```

Comment:
maybe to use UnmarshalInventory from common?

Reply:
I aligned with the code in the module. The change can be done in a separate PR across all the network utils